sonar-core/src/main/java/org/sonar/core/issue/tracking/Trackable.java

```diff
@@ -40,4 +40,6 @@
   String getLineHash();
 
   RuleKey getRuleKey();
+
+  String getStatus();
```

Comment:
This might have implications in everything that uses the normal tracker: CE, scanner, sonarlint.  Since it's only used by the SimpleTracker, why not change the SimpleTracker to use DefaultIssue? Also, what about using status(), which already exists in DefaultIssue?

Reply:
In SonarLint we are using our own copy of the issue tracking algorithm, and on Scanner side we are using DefaultIssue as implementation (and BTW local issue tracking will die sooner or later).
I created a new method (instead of reusing status()) to be consistent with other method names. This was done in a similar way for getRuleKey(), getLineHash()...

Reply:
Ok